### PR TITLE
[MFTF] use action group for go to orders grid page

### PR DESCRIPTION
**Pull request by Usik2203**
This PR use `<actionGroup ref="AdminOpenOrdersPageActionGroup" stepKey="goToOrders"/>` for navigation on Orders page instead 
`<amOnPage url="{{AdminOrdersPage.url}}" stepKey="onOrdersPage"/>`
`<waitForPageLoad stepKey="waitForOrdersPageLoad"/>`

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run all tests

**Comment by rogyar** [comment]
Hi @Usik2203. You don't need to run all tests in such cases since it might be really slow. It's enough to run the functional tests only.

File: app/code/Magento/Sales/Test/Mftf/ActionGroup/AdminOpenOrdersPageActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminOpenOrdersPageActionGroup">
```
Comment (lbajsarowicz) [inline app/code/Magento/Sales/Test/Mftf/ActionGroup/AdminOpenOrdersPageActionGroup.xml]: Wouldn't it be better to have `AdminOrdersPageOpenActionGroup`? This way all other Action Groups related to Orders Page would be close in list to each other, even in Autosuggestion in PHPStorm:

Eg. when you start to type `AdminOrderPage` and you see hints:
- `OpenActionGroup`
- `ClickFirstActionGroup`
etc. :)

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B tests

**Comment by Usik2203** [comment]
@magento run Functional Tests B2B test

**Comment by Usik2203** [comment]
@magento run Functional Tests B2B

**Review by lbajsarowicz — APPROVED** [review]
:heavy_check_mark: Perfect catch! Thank you for your input.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-7764](https://jira.corp.magento.com/browse/ENGCOM-7764) has been created to process this Pull Request

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-7764](https://jira.corp.magento.com/browse/ENGCOM-7764) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
**Notice:** QA not applicable

**Comment by engcom-Kilo** [comment]
@magento run all tests

**Comment by engcom-Kilo** [comment]
Failed Functional Tests B2B - not related to changes in this Pull Request.

**Pull request merged**